### ISSUE TITLE
Componentized Rollbar does not check for existence of polyfillJSON before use

### ISSUE THREAD

**Issue by johnnywang** [rollbar/rollbar.js#887]
**Version**: 2.19.2

It seems like there's a situation in which the custom bundled/componentized Rollbar will error out during setup due to an undefined `polyfillJSON`.

While https://github.com/rollbar/rollbar.js/pull/817 describes the JSON polyfill as needed for pre-2009 browsers that are missing it, it does seem to happen as well in the latest Chrome (and I suspect other browsers), likely when conflicting with another library on the site that overrides/wraps/redefines `JSON.stringify` and `JSON.parse` (which I assume is what the `isNativeFunction` checks for).

A check should probably be added, and/or documentation as well explaining that the polyfill component is not optional in situations where there may be library conflicts.

There's also a chance that `isNativeFunction` is not working as expected either, because on the customer site our library is loaded onto, console logging the JSON functions in Chrome claim that it is native code:

<img width="563" alt="Screen Shot 2020-08-20 at 12 19 11 PM" src="https://user-images.githubusercontent.com/938210/90815924-4c5df880-e2e0-11ea-8356-3fc268d4f168.png">

**Code Setup**
```
import Rollbar from 'rollbar/src/browser/core';

import instrumenter from 'rollbar/src/browser/telemetry';
import scrub from 'rollbar/src/scrub';
import telemeter from 'rollbar/src/telemetry';
import truncation from 'rollbar/src/truncation';

Rollbar.setComponents({
  telemeter,
  instrumenter,
  scrub,
  truncation,
});
```

**Failing Code**
```
function setupJSON(polyfillJSON) {
  if (__initRollbarJSON) {
    return;
  }
  __initRollbarJSON = true;

  if (isDefined(JSON)) {
    if (isNativeFunction(JSON.stringify)) {
      RollbarJSON.stringify = JSON.stringify;
    }
    if (isNativeFunction(JSON.parse)) {
      RollbarJSON.parse = JSON.parse;
    }
  }
  if (!isFunction(RollbarJSON.stringify) || !isFunction(RollbarJSON.parse)) {
    polyfillJSON(RollbarJSON);  // this line here does not check that polyfillJSON is defined prior to use
  }
}
```

**Comment by waltjones** [comment]
@johnnywang Thank you for the report. Will get a fix out asap.

**Comment by waltjones** [comment]
Fix released in v2.19.3.
